setup.cfg

```diff
@@ -9,6 +9,11 @@ exclude = .eggs,*.egg,build,caffe_pb2.py,caffe_pb3.py,docs,.git
 [tool:pytest]
 filterwarnings =
     error
+    # TODO(kataoka): ImportWarning, which is ignored by default, may be useful
+    # to find root causes of errors, but usually not important enough to stop
+    # testcases immediately and to hide other warnings and errors. Remove the
+    # entry below after other tests are fixed.
+    default::ImportWarning
```

Comment:
Can we filter just the `Can't resolve package from __spec__ or __package__` warning?
I think it will make the purpose of the filter self-explanatory.

Reply:
To close only chainer/chainer-test#534,
```
# importing old SciPy is warned because ...
ignore:can\'t resolve package from __spec__ or __package__, falling back on __name__ and __path__:ImportWarning:scipy
```
 is better.  The reason is probably from old Cython like the CuPy warning explained in https://github.com/chainer/chainer/pull/8131#issuecomment-532974897, while I'm not sure.  I've not found the concrete range of versions of SciPy where the warning emits.

The purpose of the PR is lowering the priority of `ImportWarning`s to be fixed.  IMO, `default::ImportWarning` is not as bad as `ignore::ImportWarning`.  What do you think?

Reply:
Your point makes total sense, I see it now :).
Sorry for the delay!